pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -80,11 +80,12 @@ class Measurements final
      * observables.
      *
      * @tparam functor_t Expectation value functor class for Kokkos dispatcher.
-     * @tparam nqubits Number of wires.
+     * @tparam num_wires Number of wires.
      * @param wires Wires to apply the observable to.
      */
     template <template <class> class functor_t, int num_wires>
-    PrecisionT applyExpValNamedFunctor(const std::vector<std::size_t> &wires) {
+    auto applyExpValNamedFunctor(const std::vector<std::size_t> &wires)
+        -> PrecisionT {
```

Comment:
Is this for style? If so, do we want to use it everywhere from now on?

Reply:
This is mainly for style and we have followed this in many Lightning src files.